src/gitea/user/base.py

```
@@ -2,72 +2,108 @@
+
+        return endpoint, kwargs
+
+    def _update_user_settings_endpoint(self) -> str:
+        """Get the endpoint for updating user settings.
+
+        Returns:
+            The API endpoint for updating user settings.
+        """
+        return "/user/settings"
+
+    def _update_user_settings_helper(  # noqa: PLR0913
```

Ruff (0.14.13)

Unused `noqa` directive (non-enabled: `PLR0913`)

Remove unused `noqa` directive

(RUF100)